Sources/App/routes.swift

```diff
@@ -86,9 +89,12 @@ func routes(_ app: Application) throws {
             app.get(":owner", ":repository", ":reference", "js", "**") {
                 try await PackageController.documentation(req: $0, fragment: .js)
             }
-            app.get(":owner", ":repository", ":reference", "theme-settings.json") {
+            app.get(":owner", ":repository", ":reference", .fragment(.themeSettings)) {
                 try await PackageController.documentation(req: $0, fragment: .themeSettings)
             }
+            app.get(":owner", ":repository", ":reference", "tutorials", "**") {
+                try await PackageController.documentation(req: $0, fragment: .tutorials)
+            }
```

Comment:
This missing route fixes the tutorial lookups. I'm not sure why it worked on first load but I'm guessing it has something to do with the JS web app.